packages/react/src/components/Skeleton/Circle/Circle.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import classes from '../Skeleton.module.css';
+import { useSynchronizedAnimation } from '../../../hooks';
+
+export type CircleProps = {
+  width?: string | number;
+  height?: string | number;
+} & HTMLAttributes<HTMLDivElement>;
+
+/**  Skeleton component used for indicating loading elements of circular shape */
```

Comment:
Since we are using compound component pattern this also has to be defined in the `index.ts` in the `SkeletonComponent` type for it to be applied properly.